syllabus.Rmd

```diff
@@ -1,64 +1,82 @@
 ---
-title: '**Baltimore Community Data Science (PH140.801) Syllabus**'
+title: '**Syllabus**'
 output:
   pdf_document: default
   html_document: default
 ---
 
+```{r setup, echo = FALSE, message = FALSE}
+library(config)
+
+knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+```
+
 <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
 
 
 <!-- <a href="syllabus.pdf" target="_blank" download = "syllabus.pdf"  ><button class="btn"><i class="fas fa-file-alt"></i> PDF version</button></a> -->
 
-
+<br>
 
 ### Course Description
 
-This course is focused on helping students learn how to apply their data science knowledge in non-academic settings. Students will partner with community-based organizations (CBOs) in Baltimore to work on a data science related goal. Students will focus on developing critical service-learning and professional skills as they work with CBOs to accomplish a data science related goal.
+This course is focused on helping students learn how to apply their data science knowledge in non-academic settings. Students will partner with [community-based organizations (CBOs)](https://source.jhu.edu/about/partner-organizations) in Baltimore to work on a data science related goal. Students will focus on developing critical service-learning and professional skills as they work with CBOs to accomplish a data science related goal. _This course can feel uncomfortable: students are expected to be self motivated and work on solutions to problems without being told what to do._
```

Comment:
maybe we should add something about how this gives a realistic experience of working in academia as a PI or working in industry or something?